web_monitoring/tests/test_db.py

```diff
@@ -292,3 +293,41 @@ def test_validate_credentials_should_raise():
     cli = Client(**bad_auth)
     with pytest.raises(UnauthorizedCredentials):
         cli.validate_credentials()
+
+
+@patch.dict('os.environ', clear=True)
+@patch('web_monitoring.db.requests.Session')
+def test_from_env_with_default_timeout(mock_session):
+    os.environ.update({'WEB_MONITORING_DB_URL': AUTH['url'],
+                       'WEB_MONITORING_DB_EMAIL': AUTH['email'],
+                       'WEB_MONITORING_DB_PASSWORD': AUTH['password']})
+    cli = Client.from_env()
+    cli.get_user_session()
+    mock_session.return_value.request.assert_called_with(
+        method='GET', url=f'{AUTH["url"]}/users/session', timeout=DEFAULT_TIMEOUT)
+
+
+@patch.dict('os.environ', clear=True)
+@patch('web_monitoring.db.requests.Session')
+def test_from_env_with_custom_timeout(mock_session):
+    os.environ.update({'WEB_MONITORING_DB_URL': AUTH['url'],
+                       'WEB_MONITORING_DB_EMAIL': AUTH['email'],
+                       'WEB_MONITORING_DB_PASSWORD': AUTH['password'],
+                       'WEB_MONITORING_DB_TIMEOUT': '7.5'})
+    cli = Client.from_env()
+    cli.get_user_session()
+    mock_session.return_value.request.assert_called_with(
+        method='GET', url=f'{AUTH["url"]}/users/session', timeout=7.5)
+
+
+@patch.dict('os.environ', clear=True)
+@patch('web_monitoring.db.requests.Session')
+def test_from_env_with_no_timeout(mock_session):
+    os.environ.update({'WEB_MONITORING_DB_URL': AUTH['url'],
+                       'WEB_MONITORING_DB_EMAIL': AUTH['email'],
+                       'WEB_MONITORING_DB_PASSWORD': AUTH['password'],
+                       'WEB_MONITORING_DB_TIMEOUT': '0'})
```

Comment:
Just realizing how repetitive this is. Worth creating a re-usable dictionary?

Reply:
You can simplify these by adding keyword args to `update`, e.g:

```py
# dict.update(other_dict, additional='key' value='pairs')
os.environ.update(AUTH, WEB_MONITORING_DB_TIMEOUT=0)
```

Reply:
Oh, nice! Learned something new.

Reply:
On second glance, I think we'd still need to create a new dict or do a fancy transformation on `AUTH`. It's setup to be spread over the constructor args, i.e. `Client(**AUTH)`. 

Now that I can delete all that code for parsing a timeout ENV variable, I'll just go back to creating a `Client` with the constructor.